tests/integration/Owncloud/OcisPhpSdk/OcisPhpSdkTestCase.php

```diff
@@ -21,6 +21,11 @@ class OcisPhpSdkTestCase extends TestCase
      */
     protected $createdDrives = [];
 
+    /**
+     * @var array <\Owncloud\OcisPhpSdk\Group>
+     */
+    protected $createdGroup = [];
```

Comment:
its an array, and obviously there cab be more that one group, so:
```suggestion
    protected $createdGroups = [];
```